widget/embedded/src/components/ConfirmWalletsModal/ConfirmWalletsModal.tsx

```diff
@@ -523,82 +468,17 @@ export function ConfirmWalletsModal(props: PropTypes) {
                     />
                   </ListContainer>
                   {!isLastWallet && <Divider size={32} />}
-                  {isLastWallet && config?.customDestination !== false && (
-                    <CustomDestination>
-                      <CustomCollapsible
-                        onOpenChange={
+                  {isLastWallet &&
+                    lastStepToBlockchain &&
+                    config?.customDestination !== false && (
```

Comment:
To improve readability, it's better to create a variable and move these checks into it:
```
const showCustomDestination = isLastWallet && lastStepToBlockchain && config?.customDestination !== false 
```

Reply:
fixed